ruby/json_object_with_array_values.rb

```diff
@@ -0,0 +1,11 @@
+# Script takes arguments and transforms them into dict with arrays as dict values
+# and returns as JSON
+require 'json'
+
+my_strings = ARGV
+
+# Make dict with the string as key and list of letters as value
+string_letters_dict = Hash[my_strings.map { |string| [string, string.chars.map(&:upcase)]}]
```

Comment:
This is really nitpicky... but we could also do `string.upcase.chars`

Reply:
Do you mean like this: `my_strings.to_h { |string| [string.upcase, string.chars.map(&:upcase)] }` so that the input is: `"" a bc def` and the output is all caps? `{"": [], "A": ["A"], "BC": ["B", "C"], "DEF": ["D", "E", "F"]}`

Reply:
I was saying `string.upcase.chars` gives the same result as `string.chars.map(&:upcase)`

Sorry-- I shouldn't have brought it up, it's too inconsequential

Reply:
Oh gotcha. No worries. Your suggestion is nicer/simpler. Will use it!